cosmic-applet-time/src/window.rs

```diff
@@ -282,7 +345,7 @@ impl cosmic::Application for Window {
                         .size(self.core.applet.suggested_size(true).0)
                         .symbolic(true),
                 );
-                for d in self.now.format("%x").to_string().split('/') {
+                for d in self.format_time(&self.now, "%x").split('/') {
```

Comment:
This doesn't work anymore because not every locale uses '/'.

Reply:
fixed